lib/internal/bootstrap/web/exposed-window-or-worker.js

```diff
@@ -19,13 +19,18 @@ const {
   defineLazyProperties,
   defineReplaceableLazyAttribute,
   exposeLazyInterfaces,
+  exposeInterface,
 } = require('internal/util');
 
 const {
   ERR_INVALID_THIS,
   ERR_NO_CRYPTO,
 } = require('internal/errors').codes;
 
+const { installObjectURLMethods, URLPattern } = require('internal/url');
+// https://urlpattern.spec.whatwg.org/#urlpattern-class
+exposeInterface(globalThis, 'URLPattern', URLPattern);
```

Comment:
I noticed that the label https://github.com/nodejs/node/labels/semver-major was removed at https://github.com/nodejs/node/pull/56452#event-15892932080. Is it intentional?

Reply:
I'll open another pull-request for making this global, rather than making this a semver-major, so we can backport it easily and land it on current release lines. This is on my todo list. I'll resolve this and remove the semver-major (once again) once I remove the global

Reply:
Removed the global, will open another PR once this lands.